media/css/newsletter/newsletter-online-harassment.scss

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.section-subscribe {
+    .mzp-l-content:first-child {
+        padding-bottom: 0;
+    }
+
+    .page-title {
+        @include text-title-lg;
+        text-align: center;
+        font-weight: 600;
+    }
+
+    .page-subtitle {
+        @include text-title-xs;
+        color: #475467;
```

Comment:
**Nit:** I think our titles are usually black in color, this one is more on the grey side?

Reply:
agree